source/resampling_with_code2.Rmd

```diff
@@ -40,62 +40,66 @@ Robert Swain was a young Black man who was sentenced to death in the early
 the eligible jurors in that county were Black, but every member of Swain's
 jury was White.  Swain and his legal team appealed to the Alabama Supreme
 Court, and then to the [US Supreme
-Court](https://en.wikipedia.org/wiki/Swain_v._Alabama), arguing there was
-that Black jurors were very rarely selected for juries in Talladega county, even though they made up about a quarter of the eligible pool of jurors.
+Court](https://en.wikipedia.org/wiki/Swain_v._Alabama), arguing
+that Black jurors were very rarely selected for juries in Talladegna county, even though they made up about a quarter of the eligible pool of jurors.
 The US Supreme Court rejected this argument, in a 6 to 3 opinion, writing that
 "The overall percentage disparity has been small and reflects no studied
 attempt to include or exclude a specified number of Negros.".
 
-The courts heard a lot of evidence on this bias in selection, but one
-observation that Swain and his team made, was that Swain's own jury was
-entirely White. To make our task simpler, we will only look at the composition
-of Swain's jury.
-
-The Supreme Court decided that the "disparity" between selection of White and
-Black jurors "has been small" — but how would they, and how would we, make a rational decision about whether this disparity really was "small"?
+And while the courts heard a lot of evidence on bias in jury selection, one observation made by Swain and his team stands out: that Swain's own jury was *entirely* White.
+The Supreme Court decided that the "disparity" between selection of White and Black jurors "has been small" — but how would they, and how would we, make a rational decision about whether this disparity *really* was "small"?
 
 ## A small disparity and a hypothetical world
 
 To answer the question that the Supreme Court asked, we return to the method we used in the last chapter.
 
-Let us imagine a hypothetical world, where each individual Black or White
-jurors had an equal chance of being selected for the jury.  Call this world Hypothetical County, Alabama.
+Let us imagine a hypothetical world, in which each individual Black or White
+person had an equal chance of being selected for the jury.  Call this world Hypothetical County, Alabama.
 
-Just as in 1960's Talladega County, 26% of the eligible jurors in Hypothetical
+Just as in 1960's Talladega County, 26% of eligible jurors in Hypothetical
 County are Black.  Hypothetical County jury selection has no bias against
-Black men, so we expect around 26% of the jury will be Black. 0.26 * 12 =
-3.12, so we expect that, on average, a little more than 3 out of 12 jurors in
+Black men, so we expect around 26% of the jury to be Black. 0.26 * 12 =
```

Comment:
The jurors were indeed restricted to men, at the time, but maybe we can miss that detail out:

```suggestion
Black people, so we expect around 26% of the jury to be Black. 0.26 * 12 =
```

Reply:
Right, the 60s! I completely forgot. Happy to change this back to be factually accurate if you want.